course7/homeworks/patrickh/ls/ls.cc

```diff
@@ -1,14 +1,99 @@
 #include <iostream>
 #include <boost/filesystem.hpp>
+#include <boost/program_options.hpp>
```

Comment:
It's always better to order the includes from the most general to the most specific. In this case the boost includes should be before iostream.